tekton/resources/nightly-tests/base/deploy_tekton_component.yaml

```diff
@@ -12,6 +12,13 @@ spec:
   - name: extra-file
     description: extra file to install (for instance, interceptors.yaml for triggers)
     default: ""
+  env:
+  - name: PACKAGE
+    value: $(params.package)
+  - name: VERSION
+    value: $(params.version)
+  - name: EXTRA_FILE
+    value: $(params.extra-file)
```

Comment:
You cannot have an `env` directly under `spec`, it must either go into `steps` or into a `stepTemplate`.